### PR TITLE
Update service-premium-what-is.md

### PR DESCRIPTION
**Pull request by minghan**
Attempt to reduce convolution in the docs.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@minghan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@minghan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@minghan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/admin/service-premium-what-is.md [hunk 1]
```diff
@@ -173,25 +173,27 @@ To learn more, see [Configure workloads in a Premium capacity](service-admin-pre
...
+Import models must be fully loaded into memory so they can be queried or refreshed. The Power BI service uses sophisticated algorithms to manage memory usage fairly, but in rare cases, the capacity can get overloaded if there are insufficient resources to meet customer's realtime demands. While it's possible for a capacity to store many import models in persistent storage (up to 100 TB per Premium capacity), not all the models necessarily reside in memory at the same time, otherwise their in-memory dataset size can easily exceed the capacity memory limit. On top of the having the dataset reside in memory, aditional memory is often required for querying and refresh.
```
Comment (amirchandani) [inline powerbi-docs/admin/service-premium-what-is.md:180]: "Besides the memory required to load the datasets, additional memory is needed for execution of queries and refresh operations."

File: powerbi-docs/admin/service-premium-what-is.md [hunk 2]
```diff
@@ -173,25 +173,27 @@ To learn more, see [Configure workloads in a Premium capacity](service-admin-pre
...
 
-It's important to stress that dataset eviction is a normal and expected behavior. It strives to maximize memory usage by loading and unloading models whose combined sizes can exceed available memory. This is by design, and transparent to report users. High eviction rates do not necessarily mean the capacity is insufficiently resourced. They can, however, become a concern if query or refresh responsiveness is suffering because of high eviction rates.
+It's important to stress that dataset eviction is a normal behavior on the capacity. The capacity strives to balance memory usage by managing the in-memory lifecycle of models in a way which is transparent to users. A high eviction rate does not necessarily mean the capacity is insufficiently resourced. It can, however, become a concern if queries performance persistently degrade due to the overhead of loading and evicting models repeatedly within a short span of time.
```
Comment (amirchandani) [inline powerbi-docs/admin/service-premium-what-is.md:186]: ...if the performance of queries or refreshes degrades due to the ...

File: powerbi-docs/admin/service-premium-what-is.md [hunk 3]
```diff
@@ -173,25 +173,27 @@ To learn more, see [Configure workloads in a Premium capacity](service-admin-pre
...
+Incremental refresh performs partition refresh instead of a full model refresh, and hence, will typically be faster and require less memory, and can substantially reduce the capacity's resource usage. Refreshes can also be CPU-intensive for models, especially those with complex Power Query transformations, or calculated tables/columns that are complex or are based on large volume of data.
+
+Refreshes, like queries, require the model be loaded into memory. If there is insufficient memory, the Power BI service will attempt to evict inactive models, and if this is not possible (as all models are active), the refresh job is queued. Refreshes are typically CPU-intensive, even more so than queries. For this reason, a limit on the number of concurrent refreshes, calculated as the ceiling of 1.5 x the number of backend v-cores, is imposed. If there are too many concurrent refreshes, the scheduled refresh is queued until a refresh slot is available, resulting in the operation taking longer to complete. On-demand refreshes such as those triggered by a user request or an API call will retry three times \[[1](#endnote-1)\]. If there still aren't enough resources, the refresh will then fail.
```
Comment (amirchandani) [inline powerbi-docs/admin/service-premium-what-is.md:192]: Should we also mention maxParallelism somewhere?

File: powerbi-docs/admin/service-premium-what-is.md (hunk 1 above)
Comment (minghan) [reply powerbi-docs/admin/service-premium-what-is.md:180]: Thanks!

File: powerbi-docs/admin/service-premium-what-is.md (hunk 2 above)
Comment (minghan) [reply powerbi-docs/admin/service-premium-what-is.md:186]: Thanks!

File: powerbi-docs/admin/service-premium-what-is.md (hunk 3 above)
Comment (minghan) [reply powerbi-docs/admin/service-premium-what-is.md:192]: I believe maxParallelism is typically 6? For most customers, it shouldn't matter.

**Comment by PRMerger14** [comment]
@minghan : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/admin/service-premium-what-is.md (hunk 3 above)
Comment (amirchandani) [reply powerbi-docs/admin/service-premium-what-is.md]: maxParallelism depends on the SKU...

**Review by amirchandani — APPROVED** [review]
Looks reasonable to me. Hopefully Kate/David can do final approval...

**Pull request merged**